test/integration/admin_orders_test.rb

```diff
@@ -30,7 +30,8 @@ class AdminOrdersTest < ActionDispatch::IntegrationTest
     click_link("Pending")
 
     assert_equal "/admin/orders/#{Order.first.id}", current_path
-    assert page.has_content?("November 19, 2015")
+    assert page.has_content?("#{Time.now.strftime("%B %d, %Y")}")
```

Comment:
Prefer single-quoted strings inside interpolations.<br>Do not use `Time.now.strftime` without zone. Use one of `Time.zone.now.strftime`, `Time.now.strftime.current`, `Time.now.strftime.in_time_zone`, `Time.now.strftime.utc`, `Time.now.strftime.getlocal`, `Time.now.strftime.iso8601`, `Time.now.strftime.jisx0301`, `Time.now.strftime.rfc3339`, `Time.now.strftime.to_i`, `Time.now.strftime.to_f` instead.